examples/transaction.py

```diff
@@ -1,10 +1,10 @@
-import berbix
 import sys
 import os
 import pprint
 import time
 
 sys.path.insert(1, os.path.join(sys.path[0], '..'))
+import berbix
```

Comment:
what happened here?

Reply:
this is necessary to import the local file path berbix installation instead of the pip project